README.md

```diff
@@ -1,204 +1,270 @@
+service. If you want, you can also use the same _virtual environment_ as
+_History_ service, no problem, but be aware of that.
+
+
+#### **Persister service on Docker**
+
+Like the _standalone mode_, the _Persister_ on Docker is identical to
+[that of the History service](#history-service-on-docker),
+paying attention only to the name of the docker image to be generated.
+In this case, it must refer to the Persister service.
 
-```shell
-virtualenv ./venv
-source ./venv/bin/activate
-```
 
-Which will create a `venv` folder in current directory and install
-basic Python packages there. The second command will set all environment
-variables so that new Python packages are installed there as well.
+### **Running the Persister service**
 
-To install all dependencies from Auth, execute the following commands.
+To run the _Persister_, after setting the environment variables, execute:
 
-```shell
-# you may need sudo for those
-apt-get install -y python-pip versiontools
-# There is a package that is not in PyPI
-pip install -r ./requirements/requirements.txt
-python setup.py install
+```bash
+$> python -m history.subscriber.persister
 ```
 
-The last command will generate a .egg file and install it into your virtual
-environment.
+********************************************************************************
+
+## **API documentation**
+
+The API documentation for this service is written as
+[API blueprints](https://apiblueprint.org/).
+To generate a simple web page from it, you must execute the commands below:
+
+```bash
+# you may need sudo for this:
+$> npm install -g aglio
+
+# static webpage
+$> aglio -i ./docs/history.apib -o ./docs/history.html
 
-#### Running the persistent service
+# serve apis locally
+$> aglio -i ./docs/history.apib -s
+```
 
-To run the Persistent, after setting the environment variables, execute:
+## **Tests**
+
+History has some automated test scripts. We use [Dredd]
+(<http://dredd.org/en/latest/>) to execute them:
 
 ```bash
-python -m history.subscriber.persister
+# you may need sudo for this:
+$> npm install -g dredd@5.1.4 --no-optional
 ```
 
-### Docker setup
+Also install the python test dependencies:
 
-Another alternative is to use **docker** to run the service. To build the
-container, from the repository's root:
+```bash
+$> pip install -r ./tests/requirements.txt
 
-```shell
-# you may need sudo on your machine:
-# https://docs.docker.com/engine/installation/linux/linux-postinstall/
-docker build -t <tag-for-persister-docker> -f docker/persister.docker .
+$> pip install codecov
 ```
-You can run the built containers using the following commands:
 
-```shell
-# may need sudo to run
-docker run -i -t <tag-for-persister-docker>
-```
\ No newline at end of file
+Run _Dredd_ automated test scripts:
+
+```bash
+$> ./tests/start-test.sh
+```
+
+Run the `pytest` module to run the coverage tests:
+
+```bash
+$> python -m pytest --cov-report=html --cov=history tests/
+```
```

Comment:
Add at the end 
`` 
# Issues and help

If you found a problem or need help, leave an issue in the main
[dojot repository](https://github.com/dojot/dojot) and we will help you!
``

Reply:
It is nice to add the documentation section too, as you mentioned in https://github.com/dojot/dojot/issues/1756.